data/strings/strings.txt

```diff
@@ -1571,50 +1571,49 @@
-    th = ก่อนที่คุณจะเริ่มต้น โปรดให้เราดาวน์โหลดแผนที่โลกลงในอุปกรณ์ของคุณ\nจำเป็นต้องใช้ %@ ของข้อมูล
-    tr = Başlamadan önce cihazınıza genel dünya haritasını indirelim.\n%@ alan gerekli.
-    uk = Перш ніж розпочати дозвольте нам завантажити мапу світу на ваш пристрій.\nВона потребує %@ даних.
-    vi = Trước khi bạn bắt đầu, hãy để chúng tôi tải bản đồ thế giới vào thiết bị của bạn.\nBản đồ này cần %@ dữ liệu.
-    zh-Hans = 在您开始使用此应用程序之前，请下载全球地图到您的设备。\n它将使用 %@ 的存储空间。
-    zh-Hant = 在您開始使用此應用程式之前，請下載全球地圖到您的裝置。\n它將使用 %@ 的儲存空間。
+    en = Before you start using the app, please download the general world overview to your device.\nIt will use %@ of storage.
+    af = Voordat u die app begin gebruik, laai die General World -oorsig op u toestel af.\nDit sal %@ van die berging gebruik.
+    ar = قبل البدء في استخدام التطبيق، يرجى تنزيل النظرة العامة على العالم العام على جهازك.\nسيستخدم %@ من مساحة التخزين.
+    az = Tətbiqdən istifadə etməyə başlamazdan əvvəl, ümumi dünya icmalını cihazınıza yükləyin.\n%@ Anbar istifadə edəcəkdir.
+    be = Перш чым пачаць выкарыстоўваць прыкладанне, калі ласка, запампуйце агляд Агульнага свету на сваю прыладу.\nЁн будзе выкарыстоўваць %@ захоўвання.
+    bg = Преди да започнете да използвате приложението, изтеглете общия преглед на света на устройството си.\nТой ще използва %@ място за съхранение.
+    ca = Abans de començar a utilitzar l’aplicació, descarregueu la visió general del món general al vostre dispositiu.\nUtilitzarà %@ d’emmagatzematge.
+    cs = Než začnete aplikaci používat, stáhněte si do svého zařízení obecný přehled světa.\nVyužije %@ úložného prostoru.
+    da = Før du begynder at bruge appen, skal du downloade den generelle verdensoversigt til din enhed.\nDet vil bruge %@ af lagerpladsen.
+    de = Bevor Sie die App verwenden, laden Sie bitte die allgemeine Weltübersicht auf Ihr Gerät herunter.\nEs wird %@ des Speicherplatzes verwenden.
+    el = Πριν αρχίσετε να χρησιμοποιείτε την εφαρμογή, παρακαλούμε κατεβάστε τη γενική επισκόπηση του κόσμου στη συσκευή σας.\nΘα χρησιμοποιήσει %@ του αποθηκευτικού χώρου.
+    es = Antes de empezar a utilizar la aplicación, descargue la visión general del mundo en su dispositivo.\nUtilizará %@ de almacenamiento.
+    et = Enne rakenduse kasutamise alustamist laadige oma seadmesse alla üldine maailma ülevaade.\nSee kasutab %@ salvestusruumi.
+    eu = Aplikazioa erabiltzen hasi aurretik, deskargatu munduko ikuspegi orokorra gailura.\n%@ Biltegiratze erabiliko du.
+    fa = قبل از شروع استفاده از برنامه ، لطفاً نمای کلی جهانی را در دستگاه خود بارگیری کنید.\nاز ٪@ ذخیره سازی استفاده می کند.
+    fi = Ennen kuin aloitat sovelluksen käytön, lataa laitteeseesi yleiskatsaus maailmasta.\nSe käyttää %@ tallennustilaa.
+    fr = Avant de commencer à utiliser l'application, veuillez télécharger la présentation générale du monde sur votre appareil.\nIl utilisera %@ de l'espace de stockage.
```

Comment:
`fr = Avant de commencer, permettez-nous de télécharger la carte générale du monde dans votre appareil.\n%@ de données sont nécessaires.`

Old translation is better